app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java

```diff
@@ -203,51 +205,53 @@ public void removeDescription(final UploadMediaDetail uploadMediaDetail, final i
 
     public class ViewHolder extends RecyclerView.ViewHolder {
 
-        @Nullable
-        @BindView(R.id.description_languages)
-        TextView descriptionLanguages;
+        TextView descriptionLanguages ;
 
-        @BindView(R.id.description_item_edit_text)
         PasteSensitiveTextInputEditText descItemEditText;
 
-        @BindView(R.id.description_item_edit_text_input_layout)
         TextInputLayout descInputLayout;
 
-        @BindView(R.id.caption_item_edit_text)
         PasteSensitiveTextInputEditText captionItemEditText;
 
-        @BindView(R.id.caption_item_edit_text_input_layout)
         TextInputLayout captionInputLayout;
 
-        @BindView(R.id.btn_remove)
         ImageView removeButton;
 
-        @BindView(R.id.btn_add)
         ImageView addButton;
 
-        @BindView(R.id.cl_parent)
         ConstraintLayout clParent;
 
-        @BindView(R.id.ll_write_better_caption)
         LinearLayout betterCaptionLinearLayout;
 
-        @BindView(R.id.ll_write_better_description)
         LinearLayout betterDescriptionLinearLayout;
 
+        private
+
         AbstractTextWatcher captionListener;
 
         AbstractTextWatcher descriptionListener;
 
         public ViewHolder(View itemView) {
             super(itemView);
-            ButterKnife.bind(this, itemView);
             Timber.i("descItemEditText:" + descItemEditText);
         }
 
         public void bind(int position) {
             UploadMediaDetail uploadMediaDetail = uploadMediaDetails.get(position);
             Timber.d("UploadMediaDetail is " + uploadMediaDetail);
 
+            descriptionLanguages = binding.descriptionLanguages;
+            descItemEditText = binding.descriptionItemEditText;
+            descInputLayout = binding.descriptionItemEditTextInputLayout;
+            captionItemEditText = binding.captionItemEditText;
+            captionInputLayout = binding.captionItemEditTextInputLayout;
+            removeButton = binding.btnRemove;
+            addButton = binding.btnAdd;
+            clParent = binding.clParent;
+            betterCaptionLinearLayout = binding.llWriteBetterCaption;
+            betterDescriptionLinearLayout = binding.llWriteBetterDescription;
+
+
```

Comment:
this all will be removed no need to store it in variable.

Reply:
use binding. and access where ever the variable was used.

Reply:
Well, you haven't considered the case for :

@Override
    public void onViewAttachedToWindow(@NonNull final ViewHolder holder) {
        super.onViewAttachedToWindow(holder);
        holder.binding.captionItemEditText.setEnabled(false);
        holder.binding.captionItemEditText.setEnabled(true);